.solhint.json

```diff
@@ -4,6 +4,7 @@
   "rules": {
     "prettier/prettier": "error",
     "var-name-mixedcase": "off",
+    "mark-callable-contracts": "off",
```

Comment:
what does this affect?

Reply:
https://github.com/protofire/solhint/blob/master/docs/rules/security/mark-callable-contracts.md